lti_consumer/static/css/student.css

```diff
@@ -0,0 +1,41 @@
+.xblock-student_view.xblock-student_view-lti_consumer h2.problem-header {
```

Comment:
This seems to be generated from the SCSS source below.  Do we need to check this file in to version control?